listenbrainz_spark/ftp/__init__.py

```diff
@@ -99,3 +100,13 @@ def _calc_sha256(self, filepath: str) -> str:
                 calculated_sha.update(byte_block)
 
         return calculated_sha.hexdigest()
+
+    def _read_sha_file(self, filepath: str) -> str:
+        """ Reads the SHA file and returns the string stripped of any whitespace and extra characters
+        """
+        with open(filepath, "r") as f:
+            sha = f.read().replace('\n', '')  # Get rid of new line characters
```

Comment:
I think these three lines can be replaced with:

sha = f.read().split(" ", 1)[0]

no?

Reply:
Makes sense, fixed it.